src/emulator/host/include/host/config.h

```diff
@@ -53,4 +53,5 @@ struct Config {
     std::string background_image = {};
     float background_alpha = 0.300f;
     int icon_size = 64;
+    std::string online_id;
```

Comment:
```suggestion
    std::string online_id = "Vita3K";
```
Not forget add it here also ^^